### ISSUE TITLE
WCAG 1.1.1: Ensures <img> elements have alternate text or a role of none or presentation (img)

### ISSUE THREAD

**Issue by zcjess** [zcjess/Ecoach-accessibility-issues#2]
#### Issue
`Ensures <img> elements have alternate text or a role of none or presentation` ([image-alt](https://accessibilityinsights.io/info-examples/web/image-alt))
#### Target application
[ECoach | STATS250 Home](https://umecoach.miserver.it.umich.edu/coach/1/home/)
#### Element path
img
#### Snippet
`<img src="/static/img/topics-img_messages/coach/_topic-img_general-default.png">`
#### How to fix
    Fix any of the following:
      Element does not have an alt attribute
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"
#### Environment
Chrome version 110.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by zcjess** [comment]
<img width="339" alt="image" src="https://user-images.githubusercontent.com/90011962/223475365-addf80cf-6805-4919-a29e-c26b5d055fa2.png">
